### DISCUSSION TITLE
How to use bob complete <shell>

### DISCUSSION THREAD

**Discussion by TiLopes** [MordechaiHadad/bob#158] [Q&A]
I've been trying to figure out how to use the code generated by bob complete zsh but just can't seem to get it working. How would I use the completion generated in zsh?

**Comment by MordechaiHadad** [comment]
What issue are you facing?
Simply following the instructions in: https://github.com/MordechaiHadad/bob#-shell-completion
Will suffice, takes around 2 minutes or less to setup
